elastic/apm-server
# Use "dynamic: runtime" instead of strict mapping for internal, TBS, and aggregation indices

## Motivation/summary



Use "dynamic: runtime" instead of strict mapping for internal, TBS, and aggregation indices. Avoid blocking ingestion on unexpected fields.

## Checklist




## How to test these changes


- Create an unexpected field (by dropping a field in the mapping / emitting a new field in apm-server) and make sure it doesn't block ingestion.

## Related issues



Closes #10808
